tests/apps/courses/test_factories_person.py

```diff
@@ -18,7 +18,7 @@ def test_factories_person_portrait(self):
         PersonFactory should be able to generate plugins with a realistic portrait
         for several languages.
         """
-        person = PersonFactory(page_languages=["fr", "en"], fill_portrait=True)
+        person = PersonFactory(extended_object__languages="fr,en", fill_portrait=True)
```

Comment:
I think this creates an akward page because its `languages` field is not coherent with its related title objects. Maybe not a big deal if the test passes...

Reply:
Actually it's a relevant question. If we use the correct way to build out the person & page, there will be no portrait for the second language (unless added manually).

I guess I could refactor these tests to reflect the fact that we don't expect the factory to generate multilingual portraits?